pom.xml

```diff
@@ -95,6 +95,12 @@
 			<version>${spring.version}</version>
 			<scope>test</scope>
 		</dependency>
+		<!-- https://mvnrepository.com/artifact/com.googlecode.json-simple/json-simple -->
+		<dependency>
+			<groupId>com.googlecode.json-simple</groupId>
+			<artifactId>json-simple</artifactId>
```

Comment:
So I'm curious: AddLinks was already using javax.json (javax.json-api in the POM). Why was google's JSON API introduced?

Reply:
To be honest, there is just a specific way I know to read/parse JSON files in Java. I just did it my way haha. If there is a way of doing it using the existing libraries, I'll use that

Reply:
After doing a bit of research, we should probably move to JSON-P in the future: https://javaee.github.io/jsonp/

Google's simple-json has had very little activity for quite a while. javax.json has gotten updates more recently, but is not available after Java 9. JSON-P seems to be the future direction of JSON in Java.

Reply:
OK, I'll create an issue for moving JSON parsing to JSON-P

Reply:
Do  you think you'd be able to switch to javax.json for this PR? I don't know how much work it would be, but I'm not too keen on having different dependencies in a project that all do pretty much the same thing. It can lead to problems when one part of the project use json-simple and another uses javax.json. I doubt the two are compatible but may have similar-looking classes and methods and that could cause some serious confusion down the road.

Reply:
Thats reasonable, I will handle that and create an issue for orthopairs/orthoinference

Reply:
If switching to javax.json is too difficult/big for this PR, I'll approve this one anyway, but there must be another PR shortly after this one (as in - work starts as soon as this one is merged) which fixes the problem (or moves _everything_ to JSON-P).

Reply:
Got rid of the other JSON package and switched to using `javax.json`